src/test/rgw/test_rgw_iam_policy.cc

```diff
@@ -732,6 +732,20 @@ TEST_F(PolicyTest, Eval7) {
 	    Effect::Pass);
 }
 
+TEST_F(PolicyTest, eval_conditions) {
+  auto p  = Policy(cct.get(), arbitrary_tenant,
+		   bufferlist::static_from_string(example1));
+  Environment e;
+
+  // evaluates statements, default allowed will be false
+  EXPECT_EQ(p.eval_conditions(e),
```

Comment:
eval_conditions function from rgw_iam_policy.cc not covered in existing unittest. So covering `eval_conditions` as sample testcase